css/cssom/serialize-values.html

```diff
@@ -140,11 +140,33 @@
       return null;
     }
 
-    function create_result(actual, expected) {
+    // A more elegant way of doing this would be better, but for now this exception list handles cases where "center" must be omitted.
+    var minimal_results = {
+        "background-position: 5% center": "5%",
+        "background-position: 0.5% center": "0.5%",
+        "background-position: -5% center": "-5%",
+        "background-position: -0.5% center": "-0.5%",
+        "background-position: 0px center": "0px",
+        "background-position: 1px center": "1px",
+        "background-position: 0.1em center": "0.1em",
+        "background-position: 0px center": "0px",
+        "background-position: -1px center": "-1px",
+        "background-position: -0.1em center": "-0.1em",
+        "background-position: left center": "left",
+        "background-position: center top": "top",
+        "background-position: center center": "center",
+        "background-position: center bottom": "bottom",
```

Comment:
Is this (and the `center top: top` above) correct? https://drafts.csswg.org/css-backgrounds/#typedef-bg-position says:

> If only one value is specified, the second value is assumed to be [center](https://drafts.csswg.org/css-backgrounds/#valdef-background-position-center).

So doesn't serializing `center bottom` to just `bottom` turn it into `bottom center`, which is the wrong position?

Reply:
The note just below mentions this:

> Note that a pair of keywords can be reordered while a combination of keyword and length or percentage cannot.

And indeed `bottom center` is equivalent to `center bottom`. That case is even covered by the test.

Reply:
Ohh, right, because bottom can't be a real position for the x axis, d'oh. Sounds good